adders/drizzle/config/adder.js

```diff
@@ -0,0 +1,355 @@
+import { categories, defineAdderConfig, generateAdderInfo } from "@svelte-add/core";
+import pkg from "../package.json";
+import { options } from "./options";
+
+export const adder = defineAdderConfig({
+    metadata: {
+        ...generateAdderInfo(pkg),
+        name: "Drizzle",
+        description: "Headless TypeScript ORM with a head.",
```

Comment:
I nabbed this from their [gh repo's description](https://github.com/drizzle-team/drizzle-orm):
![img](https://i.imgur.com/wAgYKhN.png)

although in hindsight, perhaps `Headless ORM for NodeJS, TypeScript and JavaScript` would be a better description. this can be found right below the logo in the README

Reply:
Yeah, that sounds way better!